tests/test_security_list.py

```diff
@@ -24,7 +24,10 @@
 
 class RestrictedAlgoWithCheck(TradingAlgorithm):
     def initialize(self, symbol):
-        self.rl = SecurityListSet(self.get_datetime, self.asset_finder)
+        self.rl = SecurityListSet(
+            self.get_datetime,
+            self.asset_finder,
+            load_from_directory('leveraged_etf_list'))
```

Comment:
(We discussed this in person, but answering here for posterity.)

The current test design includes a context which copies the leveraged etf directories to a tmp directory and then patches the `security_list` module with the path to the temp directory. For the tests which add test leveraged etf, the `add_security_data` then writes a file to that temp directory. The call to `load_from_directory` on each test (which was previously being done further down in the `leveraged_etf` property) is necessary to get a fresh list for those tests that exercise exclusions.

Going to add a patch to this PR to add a fixture for this suite as we had discussed.